packages/chaire-lib-backend/src/models/db/default.db.queries.ts

```diff
@@ -58,12 +58,20 @@ const create = async <T extends GenericAttributes, U>(
     tableName: string,
     parser: ((arg: T) => U) | undefined,
     newObject: T,
-    returning: string | string[] = 'id'
+    options: {
+        returning?: string | string[];
+        transaction?: Knex.Transaction;
```

Comment:
Would it be a good opportunity to add a function header that document the parameters??